modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -3,58 +3,74 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To ensure that your Mule applications always refer to a single source of truth for your API specifications, MuleSoft introduces a workflow to design, publish, and implement your API specifications.
+include::reuse::partial$beta-banner.adoc[tag="studio-asyncapi"]
+
+Ensure that your Mule applications always refer to a single source of truth for your API specifications. Design, publish, and implement your API specifications with the following workflow.
 
 . Design an API specification. +
-You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Code Editor].
+You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Text Editor].
```

Comment:
I wonder if making the link text be just the editor part might be more clear.

You can create and publish an API specification with either the xref:design-center::design-create-publish-api-visual-editor.adoc[Visual Editor] or the xref:design-center::design-create-publish-api-raml-editor.adoc[Text Editor].

Reply:
I like it! Thanks!